tests/acceptance/bind-data-test-attributes-in-components-test.js

```diff
@@ -47,23 +49,25 @@ if (!config.stripTestSelectors) {
     assert.equal(find('.test6').find('div[data-non-test]').length, 0, 'data-non-test does not exists');
   });
 
-  test('it binds data-test-* attributes with boolean values on components', function(assert) {
-    assert.equal(find('.test7').find('div[data-test-with-boolean-value]').length, 1, 'data-test-with-boolean-value exists');
-  });
+  if (GTE_EMBER_1_13) {
+    test('it binds data-test-* attributes with boolean values on components', function(assert) {
+      assert.equal(find('.test7').find('div[data-test-with-boolean-value]').length, 1, 'data-test-with-boolean-value exists');
+    });
 
-  test('it binds data-test-* attributes without values on components', function(assert) {
-    assert.equal(find('.test8').find('div[data-test-without-value]').length, 1, 'data-test-without-value exists');
-  });
+    test('it binds data-test-* attributes without values on components', function(assert) {
+      assert.equal(find('.test8').find('div[data-test-without-value]').length, 1, 'data-test-without-value exists');
+    });
 
-  test('it binds data-test-* attributes without values on block components', function(assert) {
-    assert.equal(find('.test9').find('div[data-test-without-value]').length, 1, 'data-test-without-value exists');
-  });
+    test('it binds data-test-* attributes without values on block components', function(assert) {
+      assert.equal(find('.test9').find('div[data-test-without-value]').length, 1, 'data-test-without-value exists');
+    });
 
-  test('it leaves data-test attribute without value untouched on components', function(assert) {
-    assert.equal(find('.test10').find('div[data-test]').length, 0, 'data-test does not exists');
-  });
+    test('it leaves data-test attribute without value untouched on components', function(assert) {
+      assert.equal(find('.test10').find('div[data-test]').length, 0, 'data-test does not exists');
+    });
 
-  test('it transforms data-test params to hash pairs on components', function(assert) {
-    assert.equal(find('.test11').find('div[data-test-something]').length, 1, 'data-test-something exists');
-  });
+    test('it transforms data-test params to hash pairs on components', function(assert) {
```

Comment:
I think this should be the only test that fails for Ember < 1.13?

Reply:
I think @pzuraq might be right. Using the versionCompat feature of ember-try and ember-source together has some issues at the moment, and we might only be testing against ember-source in all scenarios :scream:

Reply:
Hm, so _"best"_ solution for now might be to remove this one test for now (as well as investigating the bug and fixing it in ember-try)?

Reply:
The whole block of tests I commented out will fail for Ember <1.11 because positional params didn’t exist yet. They passed before because the transform was working (due to incorrect Ember version being detected) and changing the positional params into named params.